platform/security/certificate/security-certificate-keystoreeditor/src/main/java/org/codice/ddf/security/certificate/keystore/editor/KeystoreEditor.java

```diff
@@ -270,13 +352,16 @@ private synchronized void addToStore(String alias, String keyPassword, String st
                 Enumeration<String> aliases = jks.aliases();
                 while (aliases.hasMoreElements()) {
                     String jksAlias = aliases.nextElement();
-                    if (jks.isKeyEntry(jksAlias)) {
-                        Key key = jks.getKey(jksAlias, keyPassword.toCharArray());
-                        Certificate[] certificateChain = jks.getCertificateChain(jksAlias);
-                        store.setKeyEntry(alias, key, keyPassword.toCharArray(), certificateChain);
-                    } else {
-                        Certificate certificate = jks.getCertificate(alias);
-                        store.setCertificateEntry(alias, certificate);
+                    if (alias.equals(jksAlias)) {
```

Comment:
This check was added because otherwise if you had more than one entry in passed in jks file it would try to add them all with the same alias. This has been fixed to instead just add all the entries with the same alias as found in the passed in jks file.